ansible_collections/arista/avd/roles/eos_validate_state/tasks/hardware.yml

```diff
@@ -34,7 +34,7 @@
   assert:
     that:
       - powerSupplySlot.status == 'ok'
-    fail_msg: fan status is "{{ powerSupplySlot.status }}"
+    fail_msg: "Fan power supply status is {{ powerSupplySlot.status | replace('\"','') }}"
```

Comment:
Shouldn't this be "Power supply fan status is..."? It's power supply fan status we are checking right, not a power supply status for the fan? It's possible i misunderstood.

Reply:
Agreed, this is the status of the fan in the PSU